tox.ini

```diff
@@ -39,7 +39,16 @@ commands =
 ; Runs in: Github Actions
 ; Generates documentation with sphinx. There are other steps in the Github Actions workflow
 ; to publish the documentation on release.
-changedir = {toxinidir}/docs
+changedir = {toxinidir}
 deps = -rrequirements-docs.txt
 commands =
-    python -m sphinx.cmd.build -P -b html source build
+    python -m sphinx.cmd.build -P -b html docs/source docs/build/html
+
+[testenv:docs-multi]
```

Comment:
@nv-alaiacano , is there a way to avoid creating a second environment? Or a way to run the python command in one case and sphinx-multiversion in another case--like an env var or something?

Reply:
hm i'm not sure. it would be nice if we could cache the `.tox` directory to avoid rebuilding every time. I'll look into that.